src/types/index.tsx

```diff
@@ -42,6 +42,36 @@ export type UserDetails = {
   status: UserStatus;
 };
 
+export interface LeaveRequestData {
+  memberId: string;
+  leaveType: string;
+  startDate: Date;
+  endDate: Date;
+  reasonForLeave: string;
```

Comment:
Didn't we change this property?

Reply:
@muhammad-bilal-ali-khokhar I think we're looking at a problem where one PR is pending review while other still points out to main, causing old changes to be part of the PR too.

Reply:
Sir @Monfernape , this will not happen on my side in the future.